### PR TITLE
Document how to find disk usage by hand

### PR DESCRIPTION
**Pull request by Crell**
The particular filesystem ID here was scrambled so doesn't correspond to any particular project.

### CODE REVIEW

File: src/development/troubleshoot.md [hunk 1]
```diff
@@ -74,6 +74,35 @@ The memory usage of your container exceeds the limit, the kernel thus kills the
 * Use [sizing hints](https://docs.platform.sh/languages/php.html#php-worker-sizing-hints) to reduce the amount of PHP workers which reduces the memory footprint.
 * Upgrade your subscription on Platform.sh to get more computing resources. To do so, log into your [account](https://accounts.platform.sh) and edit the project.
 
+## Low disk space
+
+If you suspect you are running low on disk space on your application container, the easiest way to check it is to login using `platform ssh` and run the `df` command.  `df` has numerous options to tweak its output, but for just checking the available writeable space the simplest option is: `df -h -x tmpfs -x squashfs | grep -v /run/shared`
```
Comment (pjcdawkins) [inline src/development/troubleshoot.md]: If you add `-a` you'll get to see all the mounts, not just the first on each volume, but then you have to filter out more:

```sh
df -a -h -x proc -x squashfs -x tmpfs -x devpts -x sysfs | grep -v /run/shared
```

Might seem a bit ridiculous describing that as the "simplest option" (when `df` theoretically shows the necessary % on its own).

Comment (Crell) [reply src/development/troubleshoot.md]: Perhaps change it to "most direct"?  My first version used `df -haT` to get all possible data, but as @sruggier pointed out that's TMI for most people who just want to know how much disk mount  space they are using.

File: src/development/troubleshoot.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ git push platform <branch-name>
 
 ## HTTP 502 response
 
-If you receive HTTP 502 responses from your site, it means that your site is running out of workers, or the application process is crashing.  Here are the typical causes:
+If you receive HTTP 502 responses from your site, it means that your site is running out of workers or the application process is crashing.  Here are the typical causes:
```
Comment (Crell) [inline src/development/troubleshoot.md]: This additional comma is actually wrong and a comma splice. 😄   Please remove it, then go ahead and merge this PR.

Comment (Crell) [reply src/development/troubleshoot.md]: Oh, duh, I totally read that backwards for some reason.  We're good, please merge.

**Pull request merged**